frontend/scripts/componentSchemas/schemaUtils.ts

```diff
@@ -1,19 +1,120 @@
 import jsonpointer from 'jsonpointer';
+import { sortTextResourceBindings } from './languageUtils';
+import type {
+  LayoutSchema,
+  CondensedComponentSchema,
+  ExpandedComponentSchema,
+  AppFrontendVersion,
+  ComponentName,
+  PrefixedComponentName,
+  AllOfDefinition,
+} from './types';
+import type { KeyValuePairs } from 'app-shared/types/KeyValuePairs';
+
+export const allPropertyKeys = [];
```

Comment:
nit: should we add a type here to increase readability and maintainability? 😄 If I'm not mistaken, the type defaults to `any[]` now as there are no values